test/cluster/mv/test_mv_building.py

```diff
@@ -236,3 +236,100 @@ async def progress_saved():
+                    "--logger-log-level", "storage_service=debug",
+                    "--logger-log-level", "raft_topology=debug"])
+
+    s1, s2 = await manager.servers_add(2, cmdline=cmdline, auto_rack_dc="dc1")
+
+    host_id1 = await manager.get_host_id(s1.server_id)
+    host_id2 = await manager.get_host_id(s2.server_id)
+
+    cql = manager.get_cql()
+
+    await cql.run_async("CREATE KEYSPACE ks WITH replication = {'class': 'NetworkTopologyStrategy', 'dc1': 2}")
+    await cql.run_async("CREATE TABLE ks.t (pk int, ck int, v int, PRIMARY KEY (pk, ck))")
+
+    row_count = 50_000
+    await asyncio.gather(*[cql.run_async(f"INSERT INTO ks.t (pk, ck, v) VALUES ({i}, {i + 1}, {i + 2})") for i in range(row_count)])
+
+    # Topology coordinator = view building coordinator.
+    topology_coordinator = await get_topology_coordinator(manager)
+    if topology_coordinator != host_id1:
+        s1, s2 = s2, s1
+        host_id1, host_id2 = host_id2, host_id1
+    logger.info(f"Coordinator: server ID={s1.server_id}, IP={s1.ip_addr}, host ID={host_id1}")
+    logger.info(f"Target: server ID={s2.server_id}, IP={s2.ip_addr}, host ID={host_id2}")
+
+    # Prevent the target node from processing everything too early.
+    await manager.api.enable_injection(s2.ip_addr, "view_building_worker_pause_build_range_task", one_shot=False)
+
+    log = await manager.server_open_log(s1.server_id)
+    mark = await log.mark()
+
+    # Make the view building coordinator ignore the gossiper. The purpose of this
+    # is to simulate a situation when the gossiper doesn't mark a dead node as
+    # such immediately. In a scenario like that, the view building coordinator
+    # will be retrying to send a request to it.
+    await manager.api.enable_injection(s1.ip_addr, "view_building_coorindator_ignore_gossiper", one_shot=False)
+    await manager.server_stop(s2.server_id)
+
+    start = time.time()
+
+    await cql.run_async("CREATE MATERIALIZED VIEW ks.mv AS SELECT * FROM ks.t "
+                        "WHERE pk IS NOT NULL AND ck IS NOT NULL AND v IS NOT NULL "
+                        "PRIMARY KEY ((ck, pk), v)")
+
+    error = f"Work on tasks .* on replica {host_id2}:0, failed with error"
+
+    await log.wait_for(error, from_mark=mark)
+    await manager.api.disable_injection(s1.ip_addr, "view_building_coorindator_ignore_gossiper")
+    await manager.server_start(s2.server_id)
+
+    # Not needed anymore.
+    await manager.api.disable_injection(s2.ip_addr, "view_building_worker_pause_build_range_task")
+    await wait_for_view(cql, "mv", 2)
+
+    end = time.time()
+    # The duration of the view building process (in seconds, rounded up).
+    duration = int(end - start) + 1
+
+    matches = await log.grep(error, from_mark=mark)
+    match_count = len(matches)
+
+    logger.info(f"Got {match_count} matches")
+
+    # Technically, we shouldn't get more than one failure per second,
+    # but let's cut it some slack to avoid flakiness.
+    slack = 5
```

Comment:
This test is introduced in the first commit, before the rate limiter is introduced. I don't think that the test table only has one tablet, does it? Therefore, I'm not sure that the PR is bisectable. I think that the test should be moved to the next commit.